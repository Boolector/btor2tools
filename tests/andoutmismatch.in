1 zero 32
2 zero 32
3 and 1 1 2
