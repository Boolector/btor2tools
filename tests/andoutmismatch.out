*** catbtor: parse error in 'andoutmismatch.in' line 4: arguments length does not match
